src/config.js

```diff
@@ -59,5 +59,7 @@ module.exports = Object.assign({
       ]
     }
   },
-
+  billing: {
+    url: process.env.BILLING_URL || 'http://172.17.200.10:3000'
```

Comment:
can we consolidate this with the `APOLLO_CLIENT_URL`? I think they achieve the same goal just with `BILLING_URL` being a better name... @barbaraliau I'd like to open a PR against this for you for that, but for now can we just remove this and reference `APOLLO_CLIENT_URL` instead until then?